dataservice/api/phenotype/models.py

```diff
@@ -0,0 +1,27 @@
+from dataservice.extensions import db
+from dataservice.api.common.model import Base
+
+
+class Phenotype(db.Model, Base):
+    """
+    Phenotype entity.
+    :param _id: Unique id assigned by RDBMS
+    :param kf_id: Unique id given by the Kid's First DCC
+    :param created_at: Time of object creation
+    :param modified_at: Last time of object modification
+    :param external_id: Name given to Phenotype by contributor
+    :param hpo_id: hpo id
+    :param observed: whether phenotype is negative or positive
+    :param age_at_event_days: Age at the time phenotype was
+            observed, expressed in number of days since birth
+    """
+
+    __tablename__ = 'phenotype'
+    external_id = db.Column(db.Text())
```

Comment:
What would `external_id` be for a phenotype?

Reply:
i was thinking to differentiate between multiple phenotypes for the same participant. Seems like kf_id and postgres id are sufficient. i am going to remove it.